### ISSUE TITLE
Remove metrics from the gRPC client

### ISSUE THREAD

**Issue by thegridman** [aseovic/helidon#74]
### Requirements
The gRPC client module should not have a dependency on metrics (metrics will be in a new stand-alone module).

Once this is complete it should be possible to remove this dependency from helidon-grpc-client:
```
<dependency>
    <groupId>io.helidon.metrics</groupId>
    <artifactId>helidon-metrics</artifactId>
    <version>${project.version}</version>
</dependency>
```

### Tasks
Remove the metricType field from both the `ClientServiceDescriptor` and `ClientMethodDescriptor` classes as well as corresponding builders and tests.

**NOTE:** this work needs to be done on a branch and merged into the `grpc-client` branch.

**Comment by makannan** [comment]
Created pull request: https://github.com/aseovic/helidon/pull/85
